### PR TITLE
Final fixes and additions

### PR DESCRIPTION
**Pull request by steventus**

### CODE REVIEW

File: Assets/Scripts/Menus/PauseMenu.cs [hunk 1]
```diff
@@ -42,4 +52,6 @@ public void Pause()
         Cursor.lockState = CursorLockMode.None;
         pauseMenuUI.SetActive(true);
     }
+
+
```
Comment (michael-ryan) [inline Assets/Scripts/Menus/PauseMenu.cs:56]: random blank lines?

File: Assets/Scripts/Menus/SettingsMenu.cs [hunk 2]
```diff
@@ -15,16 +15,37 @@ public class SettingsMenu : MonoBehaviour
...
 
     public void SetMusicVolume(float musicVol)
-    {
-        musicMixer.SetFloat("musicVolume", Mathf.Log10(musicVol) * 20);
+    {        
         PlayerPrefs.SetFloat("musicVolume", musicVol); // Sets the value of SliderVolume to the music volume value.
+        musicMixer.SetFloat("musicVolume", Mathf.Log10(musicVol) * 20);
+
+        if (musicVol <= 0)
+            musicMixer.SetFloat("musicVolume", 0); //Avoid mathematical errors with Mathf.Log10(0);
```
Comment (michael-ryan) [inline Assets/Scripts/Menus/SettingsMenu.cs:48]: Not too sure what's happening here, but `log(0)` is undefined. Is that ok?

File: Assets/Scripts/Menus/SettingsMenu.cs [hunk 3]
```diff
@@ -34,6 +55,18 @@ public void SetSFXVolume(float sfxVol)
         PlayerPrefs.SetFloat("SFXVolume", sfxVol); // Sets the value of SliderVolume to the sound effect volume value.
         SFXMixer.SetFloat("SFXVolume", Mathf.Log10(sfxVol) * 20);
 
+        if (sfxVol <= 0)
+            SFXMixer.SetFloat("SFXVolume", 0);
+
+        PlayerPrefs.Save();
+    }
+
+    public void SetMouseSens(float _sens)
```
Comment (michael-ryan) [inline Assets/Scripts/Menus/SettingsMenu.cs:64]: no harm in naming this `SetMouseSensitivity`. We aren't going to run out of characters and it makes this function's purpose much clearer.

Comment (michael-ryan) [reply Assets/Scripts/Menus/SettingsMenu.cs:64]: Equally, the parameter can be named `sensitivity`

File: Assets/Scripts/Player/CharacterBehaviour.cs [hunk 4]
```diff
@@ -129,10 +129,13 @@ void Update()
         // Player and Camera rotation
         if (!PauseMenu.GameIsPaused)
         {
-            rotationX += -Input.GetAxis("Mouse Y") * lookSpeed;
+            //Vertical Rotation (look up and down)
+            rotationX += -Input.GetAxis("Mouse Y") * lookSpeed * SettingsMenu.mouseSens;
```
Comment (michael-ryan) [inline Assets/Scripts/Player/CharacterBehaviour.cs:133]: equivalent to `rotationX -= Input.GetAxis("Mouse Y") * lookSpeed * SettingsMenu.mouseSens;`, which is clearer to read

**Review by michael-ryan — APPROVED** [review]

File: Assets/Scripts/Menus/PauseMenu.cs (hunk 1 above)
Comment (Shadow-DA-B0SS) [reply Assets/Scripts/Menus/PauseMenu.cs:56]: Why not?

File: Assets/Scripts/Player/CharacterBehaviour.cs (hunk 4 above)
Comment (Shadow-DA-B0SS) [reply Assets/Scripts/Player/CharacterBehaviour.cs:133]: If it works, it works. Lol.

File: Assets/Scripts/Menus/SettingsMenu.cs (hunk 3 above)
Comment (Shadow-DA-B0SS) [reply Assets/Scripts/Menus/SettingsMenu.cs:64]: As long as the short hand is understandable, I think it's fine to abbreviate.

File: Assets/Scripts/Menus/SettingsMenu.cs (hunk 2 above)
Comment (Shadow-DA-B0SS) [reply Assets/Scripts/Menus/SettingsMenu.cs:48]: If it ain't broke...

**Pull request merged**